routers/web/web.go

```diff
@@ -1314,25 +1314,28 @@ func registerWebRoutes(m *web.Router) {
 	m.Group("/{username}/{reponame}", func() { // repo code
 		m.Group("", func() {
 			m.Group("", func() {
-				m.Post("/_preview/*", web.Bind(forms.EditPreviewDiffForm{}), repo.DiffPreviewPost)
```

Comment:
Why this line removed? Accidentally?

Reply:
Ouch, removed by accident indeed. I added a test for it now.